src/parkapi_sources/converters/kienzler/converter.py

```diff
@@ -9,36 +9,55 @@
 from validataclass.exceptions import ValidationError
 from validataclass.validators import AnythingValidator, DataclassValidator, ListValidator
 
-from parkapi_sources.converters.base_converter.pull import PullConverter
+from parkapi_sources.converters.base_converter.pull import PullConverter, StaticGeojsonDataMixin
 from parkapi_sources.exceptions import ImportParkingSiteException
 from parkapi_sources.models import RealtimeParkingSiteInput, SourceInfo, StaticParkingSiteInput
 
-from .models import KienzlerInput
+from .models import KienzlerGeojsonFeatureInput, KienzlerInput
 
 
-class KienzlerBasePullConverter(PullConverter):
+class KienzlerBasePullConverter(PullConverter, StaticGeojsonDataMixin):
     kienzler_list_validator = ListValidator(AnythingValidator(allowed_types=[dict]))
     kienzler_item_validator = DataclassValidator(KienzlerInput)
+    use_geojson = False
+    geojson_feature_validator = DataclassValidator(KienzlerGeojsonFeatureInput)
 
     @property
     @abstractmethod
     def config_prefix(self):
         pass
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.required_config_keys = [
-            f'PARK_API_KIENZLER_{self.config_prefix}_USER',
-            f'PARK_API_KIENZLER_{self.config_prefix}_PASSWORD',
-            f'PARK_API_KIENZLER_{self.config_prefix}_IDS',
-        ]
+    required_config_keys = [
```

Comment:
Sadly, this does not work, because `config_prefix` is just a abstract property in this moment. So `required_config_keys` is `['PARK_API_KIENZLER_<property object at 0x7935a5ba0f40>_USER', 'PARK_API_KIENZLER_<property object at 0x7935a5ba0f40>_PASSWORD', 'PARK_API_KIENZLER_<property object at 0x7935a5ba0f40>_IDS']` if you do this. The old approach setting these values at init was the correct one, because there it was set during (child) object initialization, where `config_prefix` was already set.